src/api/spec/controllers/public_controller_spec.rb

```diff
@@ -68,6 +68,7 @@
   describe 'GET #project_index' do
     context 'without view specified' do
       before do
+        package
```

Comment:
Wrongly indented, maybe?

Reply:
Oh :open_mouth:  already fixed, sorry.

Reply:
no worries - I had to run to the guardería